transport/TransportImpl.h

```diff
@@ -301,6 +303,8 @@ class TransportImpl : public RtcTransport,
     RtpSenderState& getOutboundSsrc(uint32_t ssrc, uint32_t rtpFrequency);
 
     void onTransportConnected();
+    void drainPacingBuffer(uint64_t timestamp, bool useEstimatedBudget = true);
```

Comment:
[minor] personally I am not fan of default values that can change how the method behaves. I mean, I think it is useful for things that will not affect the method output (for example: use a default value for a buffer size and add possibility to give a better value that optimizes a specific use case), otherwise seems a bit error prone

Reply:
I am totally agree with you on this one.  I checked the code prior to changes, to see whether default values are in use and found in the same TransportImpl.h (sendReports(..., bool rembReady = false)), so decided not to bring new rules into the project :) 

Changing to what I personally consider the best: non-default enum parameter that explicitly (in English) sets the mode.